search-commons/src/test/java/no/unit/nva/search/IndexDocumentGeneratorTest.java

```diff
@@ -45,17 +55,16 @@
     private static final Instant SAMPLE_PUBLISHED_DATE = Instant.now();
     private static final Map<String, String> SAMPLE_ALTERNATIVETITLES  = Map.of("a", "b","c", "d");
     private static final List<String> SAMPLE_TAGS = List.of("tag1", "tag2");
-    private static final List<String> SAMPLE_ISBNLIST = List.of("1234-5678", "98786-54321");
-    private static final Reference SAMPLE_REFERENCE = createReference();
 
-    private static Reference createReference() {
+    private static Reference createReference() throws InvalidIsbnException, InvalidUnconfirmedSeriesException {
         PublicationInstance publicationInstance = new BookMonograph.Builder().build();
-        PublicationContext  publicationContext = null;
-        try {
-            publicationContext = new Book.Builder().withIsbnList(SAMPLE_ISBNLIST).build();
-        } catch (Exception e) {
-            e.printStackTrace();
-        }
+        BookSeries series = new Series(randomPublicationChannelsUri());
+        PublicationContext  publicationContext =
+                new Book(series,
+                        randomString(),
```

Comment:
This will fail. Use the other constructor

Reply:
Changed constructor and parameternames